public_html/wp-content/plugins/wcpt/stubs/page/code-of-conduct-online.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/** @var WordCamp_New_Site $this */
+
+?>
```

Comment:
It looks like `$this` isn't used in either file, so technically this could be removed.

Reply:
Good call. e0f0726e